### ISSUE TITLE
Flow on C++

### ISSUE THREAD

**Issue by Nic9407** [xioTechnologies/NGIMU-C-Cpp-Example#3]
Hello, I'm trying to use the code in C++ and wifi communication. Once received the data from the UDP socket (return 164 bytes, so it seems it's reading something) I call the NgimuReceiveProcessUdpPacket function. It runs with no error but never get to the callback of the ProcessSensor function and so never output any data. Can you provide an example call or usage of the NgimuReceive.c for this use case? I feel I'm missing some understanding on how it should work.

Best,

**Comment by xioTechnologies** [comment]
An example call would be:
```
udpPacket = "#bundle\0\0\0\0\0\0\0\0\0/quaternion\0,fff\0\0\0\0\x00\x00\x80\x3F\x00\x00\x00\x00\x00\x00\x00\x00"; // UDP packet received from NGIMU

NgimuReceiveProcessUdpPacket(udpPacket, sizeof(udpPacket));
```
The quaternion packet hardcoded above was created as per documentation but the above code has not been tested.  Please let me know if it does not work and I will create a code project to debug.

**Comment by Nic9407** [comment]
Hello,
Thanks for the fast reply. I've found the main issue, which was the size of the buffer I was using to receive from the UDP. When that exceed 1472 the code does not decode the oscpackets. I've selected initialize an oversize buffer for the actual stream of info thus was never basically entering the sequence of callback functions

Now everything is running nice and smoothly. 
Thanks a lot for the help
